src/aleph/web/controllers/p2p.py

```diff
@@ -11,20 +16,122 @@
 LOGGER = logging.getLogger("web.controllers.p2p")
 
 
+async def get_user_usage(address: str) -> float:
+    # ETH
+    LOGGER.info("get_user_usage")
+    messages = [
+        msg async for msg
+        in Message.collection.find({
+            'content.address': address,
+            'type': 'STORE',
+        },
+            limit=10000,
+            # skip=pagination_skip,
+            sort=[('time', -1)]
+        )
+    ]
+    LOGGER.info("MESSAGES count {}".format(len(messages)))
+
+    # TODO: Check 'sender' is allowed (VM runner, ...)
+
+    usage = sum(
+        max(message['content'].get('size', 0), 0)
+        for message in messages
+    )
+    assert usage >= 0
+    return usage
+
+
+async def get_user_balance(address: str) -> float:
+    # TODO: Support ALEPH on chains other than ETH
+    # TODO: Check if using the freekey api key is acceptable
+    API_url = f"https://api.ethplorer.io/getAddressInfo/{address}?apiKey=freekey"
```

Comment:
This API may be deprecated. We should call an Aleph VM that takes care of fetching the balance across chains.